### PR TITLE
Mohamed m1 @MMGGYY66

### PR DESCRIPTION
**Pull request by MMGGYY66**
## Team member 1  @MMGGYY66
- [Create Book class in book.rb](https://github.com/alvaroiporre/Ruby-Capstone/commit/2da69b190dfddcbebfc046bf747dfc4803c1eb54)
- [Create Label class in label.rb](https://github.com/alvaroiporre/Ruby-Capstone/commit/4933423221e83b2b3716d149214c01fa0ced90f9)
- [Create unit test for the book class](https://github.com/alvaroiporre/Ruby-Capstone/commit/df779d8e40844fa246bb0b7275377fb8bc54c72e)
- [Create unit test for the Label class](https://github.com/alvaroiporre/Ruby-Capstone/commit/501b98fc2e5fd903b8e0e2c60461e7c8233ed23d)
- [Create schema.sql and add table book to it](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/16bd4b4142801958c841c87cfc21e9aa8d313a27)
- [and add table label to schema.sql ](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/c473303ea3ec40871bfa6dd0bba9e27e92d3ff1d)
- [Create list_all_books method in app.rb](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/512101bcd10e811a67c68867fb688268d42c703c)
- [Create list_all_labels method in app.rb](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/515d38755975d314275823f23415e5f4594a03d9)
- [Fix linters issues](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/9131987d3263ae83ef26b83d2860d88d8b8603d6)
- [Preserve data from Book and Label in storage class](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/22282d152c3cdfc7155453fb442d86fd71248e09)
- [update app.rb to include the storage class](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/232aed224347394149db8068d562f3a0eea13e72)
- [Create add_item_book method](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/d6b2c513c7c253dbde8952b87f144d08422cc397)
- [Create add_label method](https://github.com/alvaroiporre/Ruby-Capstone/pull/31/commits/d951b85900b4110d435474d191060d41bd2dc49d)

### CODE REVIEW

File: label.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  def initialize(title, color)
+    @id = Random.rand(0..1000)
+    @items = []
+    @title = title
+    @color = color
+  end
+
+  def add_item(item)
+    @items << item if @items.include?(item) == false
```
Comment (ezekielgn) [inline label.rb:14]: Here you could use unless instead of checking if condition is false.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There are not books created yet'
+    else
+      @books.each do |book|
+        print "Title: '#{book.label.title}, Author: '#{book.author.first_name}'"
+        puts ''
```
Comment (ezekielgn) [inline app.rb:20]: I think it would be better to use puts instead of print, or you can use \n at the end of print if you want to add extra space.

**Review by ezekielgn — CHANGES_REQUESTED** [review]
Great work, the tests are passing and everything looks good but I noticed the following things that can be improved.

File: label.rb (hunk 1 above)
Comment (MMGGYY66) [reply label.rb:14]: Both of them work well. So if it works well, no need to change it. So I think it is optional.

File: app.rb (hunk 2 above)
Comment (MMGGYY66) [reply app.rb:20]: Also. Both of them work well. So if it works well, no need to change it. So I think it is optional.

**Comment by MMGGYY66** [comment]
Done

**Comment by ezekielgn** [comment]
Great work, your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
* No linter errors ✔️
* Correct GitFlow ✔️
* Good implementations ✔️
* Optional suggestions

**Review by ezekielgn — APPROVED** [review]

**Comment by MMGGYY66** [comment]
thanks

**Pull request merged**